src/plugins/postal-savings-rs/preferences.xml

```diff
@@ -19,6 +19,16 @@
         negativeButtonText="Cancel"
         summary="||***********"
     />
+    <EditTextPreference
+        key="cardNumber"
+        obligatory="false"
+        inputType="text"
```

Comment:
Is it Android's [inputType](https://developer.android.com/reference/android/widget/TextView#attr_android:inputType)?

Reply:
Kind of. Supported types: text, textPassword, textEmailAddress, phone, number, numberDecimal, numberPassword